CMakeLists.txt

```diff
@@ -151,7 +151,9 @@ if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
     )
 
     include(CTest)
-    add_subdirectory(tests)
+    set(OTF2XX_ENABLE_ALL_TESTS ON)
 else()
     set_target_properties(otf2xx-reader PROPERTIES EXCLUDE_FROM_ALL TRUE)
 endif()
+
+add_subdirectory(tests)
```

Comment:
Because of the tight coupling of otf2xx to a specific interface version of OTF2, I don't think users should be forced to compile those tests. In fact, we never had them fail in the past five years. Running those tests during development is sufficient.

Reply:
Those tests are compiled as static libraries (I wish I could compile them into OBJECT libraries only...) which is very cheap. Given the importance of those tests I'd compile them anyway. Maybe some platform dependent type was used by accident somewhere which would go unnoticed during development.   
Still remove them?